glib/Object.cs

```diff
@@ -95,8 +95,6 @@ public virtual void Dispose ()
 				Console.WriteLine ("Exception while disposing a " + this + " in Gtk#");
```

Comment:
Can we get a description of what was broken, why it was broken and how this fixes it? Future alan will be glad that the information was documented in the commit message ;)